players/base.py

```diff
@@ -146,7 +146,10 @@ def _target_specific_id(self, target_id: int, direction: str = "l"):
                     xbox.tap_down()
 
     def attack(
-        self, target_id: Optional[int] = None, direction_hint: Optional[str] = "u"
+        self,
+        target_id: Optional[int] = None,
+        direction_hint: Optional[str] = "u",
+        record_results:bool=False
```

Comment:
If record_results is true, it will tap confirm a few times and then return so that the parent can record the damage values out of memory. Otherwise it taps confirm until it's another character's turn or the battle completes. That could probably use some love anyway, but in the meantime I'm open to suggestions.

Reply:
suggestions on the variable name that is. ;)